packages/api-bindings/src/apollo/cache/createContentPublicationTypePolicy.ts

```diff
@@ -100,10 +104,73 @@ const hasCollectedByMe = (existing: boolean, { readField }: FieldFunctionOptions
   return collectPendingTx !== undefined;
 };
 
+const isMirroredByMe = (existing: boolean, { readField }: FieldFunctionOptions): boolean => {
+  if (existing === true) return existing;
+
+  const profileIdentifier = activeProfileIdentifierVar();
+  const publicationId = readField('id') as PublicationId;
+  const mirrors = readField('mirrors') as PublicationId[];
+
+  if (!profileIdentifier) return existing;
+  if (mirrors.length > 0) return true;
+
+  const isMirrorTransactionForThisPublication = isMirrorTransactionFor({
+    publicationId,
+    profileId: profileIdentifier.id,
+  });
+
+  const mirrorPendingTxs = getAllPendingTransactions().filter((transaction) => {
+    return isMirrorTransactionForThisPublication(transaction);
+  });
+
+  return mirrorPendingTxs.length > 0;
+};
+
+const stats: FieldReadFunction<PublicationStats> = (existing, { readField }) => {
+  if (!existing) return existing;
+
+  const profileIdentifier = activeProfileIdentifierVar();
+  const publicationId = readField('id') as PublicationId;
+
+  if (!profileIdentifier) return existing;
+
+  // mirror
+  const isMirrorTransactionForThisPublication = isMirrorTransactionFor({
+    publicationId,
+    profileId: profileIdentifier.id,
+  });
+
+  const mirrorPendingTxs = getAllPendingTransactions().filter((transaction) => {
+    return isMirrorTransactionForThisPublication(transaction);
+  });
+
+  // collect
+  const isCollectTransactionForThisPublication = isCollectTransactionFor({
+    publicationId,
+    profileId: profileIdentifier.id,
+  });
+
+  const collectPendingTx = getAllPendingTransactions().find((transaction) => {
+    return isCollectTransactionForThisPublication(transaction);
+  });
+
+  return {
+    ...existing,
+    totalAmountOfMirrors: existing.totalAmountOfMirrors + mirrorPendingTxs.length,
+    totalAmountOfCollects: existing.totalAmountOfCollects + (collectPendingTx ? 1 : 0),
```

Comment:
Can you please clarify why this is not using the count?

Reply:
not, they behave the same, more than one collect in progress can exist for the same publication.